src/main/scala/com/github/unchama/seichiassist/util/Util.scala

```diff
@@ -35,6 +35,7 @@ object Util {
     player.sendMessage(RED.toString + "再接続しても改善されない場合はお問い合わせフォームまたは整地鯖公式Discordサーバーからお知らせ下さい")
   }
 
+  @Deprecated
```

Comment:
```suggestion
  @deprecated("please use ManagedWorld#isSeichiSkillAllowed")
```

Scalaにおける非推奨アノテーションは小文字です。
nits: 理由を書いておくと親切かもしれない